spec/client_spec.rb

```diff
@@ -43,7 +50,8 @@
         :userData => 'userData1',
         :clientId => client_id1,
         :online => true,
-        :at => Time.now.to_i * 1000
+        :at => Time.now.to_i * 1000,
```

Comment:
remove :at from everywhere. This is no longer used. You can also remove the tests related to "at"